pyramid_zipkin/logging_helper.py

```diff
@@ -80,9 +79,9 @@ def log_spans(self):
 
             end_timestamp = time.time()
             log_service_span(self.zipkin_attrs, self.start_timestamp,
-                             end_timestamp, self.request_path,
-                             self.request_path_qs, self.endpoint_attrs,
-                             self.request_method, self.registry_settings)
+                             end_timestamp, self.binary_annotations_dict,
```

Comment:
What do you mean, exactly? You mean `log_service_span` should just take a `dict` of param values? I don't really like that because we lose any sense of schema. With this, you at least know what you have to pass in.